build-system/tasks/OWNERS

```diff
@@ -6,5 +6,9 @@
     {
       owners: [{name: 'ampproject/wg-infra'}],
     },
+    {
+      pattern: 'check-types.js',
+      owners: [{name: 'ampproject/wg-performance'}]
```

Comment:
Should this list contain `wg-infra` as well? Or is ownership implied due to the previous rule?

Reply:
Ownership is always additive :)